src/popover/Popover.js

```diff
@@ -410,6 +411,36 @@ export default class Popover extends React.Component {
         anchorOffset.left = isHorizontal ? (child.width - popover.width) : 0;
         anchorOffset.top = isVertical ? (child.height - popover.height) : 0;
         break;
+      case 'auto':
+        _realAnchor = 'start';
```

Comment:
Yep, cause we should add logic to handle the case where it overflows the left side of the body.